source/alloc/src/heap.rs

```diff
@@ -85,6 +85,8 @@ static OOM_WAITER: WaitQueue = WaitQueue::new();
 /// that *could* potentially succeed
 static INHIBIT_ALLOC: AtomicBool = AtomicBool::new(false);
 
+pub static OOM_PANIC: AtomicBool = AtomicBool::new(false);
```

Comment:
Do we ever set this true?

Reply:
agh, this was a temporary debugging thing, i meant to remove it --- good catch.